### PR TITLE
Implemented start game command

### PR DESCRIPTION
**Pull request by TiagoFar78**
Implemented a command that allows the game to be started. I also added a base for commands execution and a GameManager,

### CODE REVIEW

File: src/main/java/net/tiagofar78/prisonescape/commands/StartSubcommand.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+			return true;
+		}
+		
+		if (args.length != 0) {
+			sender.sendMessage(messages.getStartCommandUsage());
+			return false;
+		}
+		
+		int returnCode = GameManager.startNewGame();
+
```
Comment (iquelli) [inline src/main/java/net/tiagofar78/prisonescape/commands/StartSubcommand.java:26]: remove this space

File: src/main/java/net/tiagofar78/prisonescape/commands/StartSubcommand.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		
+		int returnCode = GameManager.startNewGame();
+		
+		if (returnCode == -1) {
+			sender.sendMessage(messages.getGameAlreadyStartedMessage());
+			return false;
+		}
+		
+		sender.sendMessage(messages.getSuccessfullyStartedGameMessage());
+
```
Comment (iquelli) [inline src/main/java/net/tiagofar78/prisonescape/commands/StartSubcommand.java:33]: I-d also remove this one

**Review by iquelli — CHANGES_REQUESTED** [review]

File: src/main/java/net/tiagofar78/prisonescape/commands/PrisonEscapeCommand.java [hunk 3]
```diff
@@ -1,15 +1,75 @@
...
+
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandExecutor;
 import org.bukkit.command.CommandSender;
 
+import net.tiagofar78.prisonescape.managers.ConfigManager;
+import net.tiagofar78.prisonescape.managers.MessageLanguageManager;
+
 public class PrisonEscapeCommand implements CommandExecutor {
+
```
[the remark this answers is not in the document]
Comment (TiagoFar78) [reply src/main/java/net/tiagofar78/prisonescape/commands/PrisonEscapeCommand.java:13]: I feel like it looks cleaner if there is a line after class definition.

**Review by iquelli — APPROVED** [review]
LGTM

**Pull request merged**